arcgis-ios-sdk-samples/Augmented reality/Explore scenes in flyover AR/README.md

```diff
@@ -0,0 +1,45 @@
+# Explore scenes in flyover AR
+
+Use augmented reality (AR) to quickly explore a scene more naturally than you could with a touch or mouse interface.
+
+![Scene shown in an app](image1.png)
+
+## Use case
+
+You can use AR to drop into an area and visualize information, like a proposed development or a historical model of a city. You could use flyover AR to explore a city by walking through it virtually.
+
+## How to use the sample
+
+When you open the sample, you'll be viewing the scene from above. You can walk around, using your device as a window into the scene. Try moving vertically to get closer to the ground. Touch gestures which 
+
+## How it works
+
+1. Create the `ArcGISARView` and add it to the view.
+2. Listen for ARKit tracking state updates using `arView.arSCNViewDelegate` and provide guidance to the user as necessary.
+3. Create the scene, add content, then display it.
+4. When the content you want to view loads, get its center point and use that to create the origin camera for the AR view. Note that the altitude should be set so that all scene content is visible. For a city, a good value might be a bit higher than the tallest building. The sample uses 600 meters in the absence of tall buildings in the sample data.
+5. Set the translation factor so that you can move through the scene easily. With a translation factor of 1000, you will move 1000 feet in the scene for every foot you move the physical device.
+6. Set the space effect to `stars` and atmosphere effect to `realistic` to create an immersive experience.
+
+## Relevant API
+
+* ArcGISARView
+* SceneView
+
+## About the data
+
+This sample uses a sample [integrated mesh layer](https://www.arcgis.com/home/item.html?id=d4fb271d1cb747e696bb80adca8487fa) provided by [Vricon](https://www.vricon.com/). The integrated mesh layer shows an area around the US-Mexico border.
```

Comment:
Don't forget to update to the new item: https://www.arcgis.com/home/item.html?id=dbc72b3ebb024c848d89a42fe6387a1b